cpu-hog/prow_run.sh

```diff
@@ -23,7 +23,7 @@ setup_arcaflow_env "$SCENARIO_FOLDER"
 checks
 
 # Substitute config with environment vars defined
-export SCENARIO_FILE="$SCENARIO_FOLDER/input.yaml"
+export SCENARIO_FILE="scenarios/arcaflow/cpu-hog/input.yaml"
```

Comment:
Thats weird that you had that issue since we are setting the file using the SCENARIO_FOLDER variable

`envsubst < node-cpu-hog/input.yaml.template> $SCENARIO_FOLDER/input.yaml.template`